.github/workflows/formatting-and-tests.yml

```diff
@@ -30,18 +36,29 @@ jobs:
         strategy:
             max-parallel: 4
             matrix:
-                dotnet-version: ["3.1", "5.0", "6.0"]
```

Comment:
our project's target-framework is set to 3.1, if we set the matrix , its will work for only 3.1 but in iteration for 5.0 and 6.0 our projects wont be build because they are targeting 3.1 and it wont find framework 3.1  at the time of iteration for 5.0 and 6.0.That's why i removed it from matrix.

Reply:
Looking at the versions, should we in fact be targeting 6.0.x instead though? Based on the end of life dates [here](https://versionsof.net/) it seems that we should be. Is that going to be a massive amount of work, however?